src/components/NewsArticle/NewsArticle.module.css

```diff
@@ -0,0 +1,49 @@
+.articleBox {
+  width: 1138px;
+  height: 224px;
+  justify-content: center;
+  align-items: center;
+  padding: 30px 0px 20px 60px;
+  margin: 30px;
+
+  background: #ffffff;
+
+  box-shadow: 0px 12px 32px rgba(0, 0, 0, 0.05);
+  border-radius: 10px;
+}
+
+.articleTitle {
+  font-size: 33px;
+  line-height: 40px;
+  text-decoration: none;
+  color: #000000;
+}
+
+.publicationDateBox {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 7px 10px;
+
+  position: static;
+  width: 104px;
+  height: 35px;
+  left: 0px;
+  top: 158px;
+
+  background: rgba(236, 178, 46, 0.38);
+  border-radius: 12px;
+
+  /* Inside Auto Layout */
+
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 50px 0px;
+}
+
+.publicationDateText {
+  font-size: 14px;
```

Comment:
Make sure the text can't wrap and allow the box to expand to accommodate so we don't get wrapped dates like this:
![image](https://user-images.githubusercontent.com/13375042/137375334-feb03741-48e2-4b2c-b9a2-065123763eb9.png)